shared/reducers/chat.js

```diff
@@ -164,9 +164,7 @@ function reducer (state: Constants.State = initialState, action: Constants.Actio
     }
     case 'chat:appendMessages': {
       const appendAction: Constants.AppendMessages = action
-      const appendMessages = appendAction.payload.messages
-      const isSelected = action.payload.isSelected
-      const conversationIDKey = appendAction.payload.conversationIDKey
+      const {messages: appendMessages, isSelected, conversationIDKey, isAppFocused} = appendAction.payload
```

Comment:
this is an unrelated action to the actual focusChanged one so i figured it should be more self similar (isSelected)